rmats.py

```diff
@@ -369,6 +375,9 @@ def process_counts(istat, tstat, counttype, ase, cstat, od, tmp, stat,
     # Combine into final output
     subprocess.call(['paste', ostat_fdr, ostat_il,], stdout=resfp)
     subprocess.call([python_executable, join_2f, efn, resfp.name, '0', '0', finfn,], stdout=FNULL)
+    append_individual_counts(indiv_counts_file_name, finfn,
```

Comment:
This function includes the filtering based on `imbalance_ratio`. The filtering may be more appropriate to do before running the stats model so that the FDR correction takes the filtering into account

Reply:
The filtering has been moved to before the stats model